message-hub/README.md

```diff
@@ -0,0 +1,174 @@
+# Using IBM Message Hub and Apache Kafka to copy documents from Cloudant to Db2 Warehouse on Cloud
+
+This tutorial will use Apache Kafka Connect to publish messages from a Cloudant
+database to an IBM Cloud Message Hub topic. The messages will then be consumed by
+Apache Kafka Connect's JDBC connector to write to a Db2 Warehouse on Cloud table.
+
+Note that this example uses the Kafka Connect standalone worker configuration.
+
+**N.B. This is an example only intended as a _starting point_ for using IBM Message Hub and Apache Kafka Connect to extract data from Cloudant and insert into Db2 Warehouse on Cloud. It is _not_ intended as a drop-in replacement for the deprecated Cloudant warehouse integration. In particular this example does not create or alter Db2 tables or handle document deletion.**
+
+# Setting up IBM Cloud services
+
+## Cloudant
+
+1. [Create an IBM Cloudant](../cloudant/create.md) instance.
+1. [Replicate the tutorial data](../cloudant/replicate.md).
+
+## Message Hub
+
+1. Create an [IBM Message Hub](https://console.bluemix.net/catalog/services/message-hub) service instance.
+![Message Hub in catalog](mh_catalog.png)
+![Create Message Hub instance](mh_create.jpg)
+1. Create Service Credentials for IBM Message Hub.
+![Create Message Hub service credentials](mh_creds.jpg)
+1. Use the Message Hub `Manage` interface to create a topic with a single partition, e.g. `exampleTopic`.
+![Create Message Hub topic](mh_topic.jpg)
+
+Or alternatively use the `api_key` and the `kafka_admin_url` from your service credentials to create the topic from the command line e.g.
+```sh
+curl -v -H 'Content-Type: application/json' -H 'Accept: */*' -H 'X-Auth-Token: ${api_key}' -d '{ "name": "exampleTopic", "partitions": 1 }' <kafka_admin_url>/admin/topics
+```
+
+## Db2 Warehouse on Cloud
+
+1. [Create an IBM Db2 Warehouse on Cloud](../db2/create.md) instance.
+1. [Create a Db2 table to house the data](../db2/animaldb_table.md).
+When using your own data be sure to choose an appropriate table schema and
+consider the specifics of JSON to Struct conversion as documented by
+ `kafka-connect-cloudant` (e.g. numbers represented as FLOAT64/double).
+1. From the Db2 Warehouse dashboard select the `JDBC` in the connect dropdown
+and follow the link to the connection information.
+![JDBC connect dropdown](db2_conn.jpg)
+1. Follow the instructions to install the Db2 JDBC driver for your platform if needed.
+![Install JDBC driver](db2_jdbc_driver.jpg)
+1. Copy ther JDBC connection string as you will need it to configure the
+kafka-connect-jdbc sink connector.
+![JDBC connection string](db2_jdbc.jpg)  
+
+# Setting up Apache Kafka Connect
+
+1. Download [Apache Kafka](https://kafka.apache.org/downloads) - this tutorial used Apache Kafka version 1.0.0 for Scala 2.11.0 (`kafka_2.11-1.0.0.tgz`).
+1. Extract the Apacke Kafka installation.
+1. Create a `connect-standalone.properties` file.
```

Comment:
Since all the other .properties files are under the config dir, it's probably good to add at the end `...under the config directory.`

Reply:
Done in 8f427dd61e5c4025362475c422b517c5ce28338b